app/src/ui/app.tsx

```diff
@@ -826,6 +826,7 @@ export class App extends React.Component<IAppProps, IAppState> {
             key="delete-branch"
             dispatcher={this.props.dispatcher}
             repository={popup.repository}
+            account={null}
```

Comment:
Because I forgot to pass the account. `null` is simply a place holder used to make sure it worked when no account is passed in.